ad7768-1fmcz/ad77681.c

```diff
@@ -367,6 +632,297 @@ int32_t ad77681_set_mclk_div(struct ad77681_dev *dev,
 	return ret;
 }
 
+/**
+ * Set the VCM output.
+ * @param dev - The device structure.
+ * @param VCM_out - The VCM output voltage.
+ * 					Accepted values: AD77681_VCM_HALF_VCC
+ *									 AD77681_VCM_2_5V
+	  								 AD77681_VCM_2_05V
+									 AD77681_VCM_1_9V
+									 AD77681_VCM_1_65V
+									 AD77681_VCM_1_1V
+									 AD77681_VCM_0_9V
+									 AD77681_VCM_OFF
+ * @return 0 in case of success, negative error code otherwise.
+ */
+int32_t ad77681_set_VCM_output(struct ad77681_dev *dev,
+			       enum ad77681_VCM_out VCM_out)
+{
+	int32_t ret;
+
+	ret = ad77681_spi_write_mask(dev,
+				     AD77681_REG_ANALOG2,
+				     AD77681_ANALOG2_VCM_MSK,
+				     AD77681_ANALOG2_VCM(VCM_out));
+
+	if (ret == SUCCESS)
+		dev->VCM_out = VCM_out;
+
+	return ret;
+}
+
+/**
+ * Set the AIN- precharge buffer.
+ * @param dev - The device structure.
+ * @param AINn - The negative analog input precharge buffer selector
+ * 					Accepted values: AD77681_AINn_ENABLED
```

Comment:
nitpick: This header comment looks messy. Check the entire file for similar issues.

Reply:
Done.